lib/content-services/src/lib/viewer/components/alfresco-viewer.component.spec.ts

```diff
@@ -350,6 +353,12 @@ describe('AlfrescoViewerComponent', () => {
         expect(component.nodeId).toBe('id1');
     }));
 
+    it('should download file when downloadFile event is emitted', () => {
+        spyOn(nodeActionsService, 'downloadNode');
+        component.onDownloadFile();
+        expect(nodeActionsService.downloadNode).toHaveBeenCalled();
```

Comment:
This unit test is not really testing anything we can be sure that if you call `onDownloadFile` downloadNode will be called is the only instruction inside

Reply:
We could have a mock component that could emit the downloadFile event, which would then be linked with this method and then update the test case according to that, but I feel that then we would essentially be writing a test case for event handling in Angular, and won't really add anything new to test the feature we are implementing here. 

Do you recommend removing this test case altogether in that case?

Reply:
I would say calling `component.onDownloadFile()` is a problem, as you are not testing the component behaviour, but just a method... there's no guarantee that this method is called by the UI. So better testing would be actually invoking the component behaviour that results in the `nodeActionsService.download` call

Reply:
Updated the test case to test service call by actually emitting the downloadFile event from ViewerComponent